src/magic_folder/test/cli/test_magic_folder.py

```diff
@@ -589,143 +588,366 @@ def addOutcomeDetails(testcase, outcome):
     )
 
 
-class CreateMagicFolder(MagicFolderCLITestMixin, AsyncTestCase):
-    def test_create_and_then_invite_join(self):
-        self.basedir = "cli/MagicFolder/create-and-then-invite-join"
-        self.set_up_grid(oneshare=True)
-        local_dir = os.path.join(self.basedir, "magic")
-        os.mkdir(local_dir)
-        abs_local_dir_u = abspath_expanduser_unicode(unicode(local_dir), long_path=False)
+class CreateMagicFolder(AsyncTestCase):
+    @defer.inlineCallbacks
+    def setUp(self):
+        """
+        Create a Tahoe-LAFS node which can contain some magic folder configuration
+        and run it.
+        """
+        yield super(CreateMagicFolder, self).setUp()
+        self.client_fixture = SelfConnectedClient(reactor)
+        yield self.client_fixture.use_on(self)
 
-        d = self.do_create_magic_folder(0)
-        d.addCallback(lambda ign: self.do_invite(0, self.alice_nickname))
-        def get_invite_code_and_join(args):
-            (rc, stdout, stderr) = args
-            invite_code = stdout.strip()
-            return self.do_join(0, unicode(local_dir), invite_code)
-        d.addCallback(get_invite_code_and_join)
-        def get_caps(ign):
-            self.collective_dircap, self.upload_dircap = self.get_caps_from_files(0)
-        d.addCallback(get_caps)
-        d.addCallback(lambda ign: self.check_joined_config(0, self.upload_dircap))
-        d.addCallback(lambda ign: self.check_config(0, abs_local_dir_u))
-        return d
+        self.tempdir = self.client_fixture.tempdir
+        self.node_directory = self.client_fixture.node_directory
+
+    @defer.inlineCallbacks
+    def test_create_magic_folder(self):
+        # Get a magic folder.
+        magic_folder = self.tempdir.child(u"magic-folder")
+        outcome = yield cli(
+            self.node_directory, [
+            b"create",
+            b"magik:",
```

Comment:
personally, i would indent these (in the list) another level so it's more obvious they're "parts of a single arg"  .. but maybe that's just a style thing :)

Reply:
I could get behind running "black" on the whole codebase and the enforcing it all in CI, I think.

Reply:
Fixed the formatting and added docstrings in the commits 0f316de and 
9c4ac61. Thanks.